test/index.js

```diff
@@ -650,7 +651,7 @@ describe('mdast.process(value, options, done)', function () {
     it('should run plugins', function () {
         assert(
             mdast.use(mentions).process('@mention') ===
-            '[@mention](https://github.com/blog/821)\n'
+            '[@mention](https://github\\.com/blog/821)\n'
```

Comment:
Oh my mistake, I remember it was either really early or really late when I posted that haha, sorry!